request_a_repository.md

```diff
@@ -0,0 +1,13 @@
+# How to Request a New Repository
+
+To request a new repository in the Operate First organization, you must open an issue in the [common repository][1].
+
+1. Select **New issue**
+1. Click **Get started** next to Repository creation
+1. Fill out the issue template
+1. Select **Submit new issue**
+
+If you have any questions or need additional help, please post a message in the `#support` channel on the [Operate First slack](2).
+
+[1]: https://github.com/operate-first/common/issues
+[2]: https://app.slack.com/client/T01SKMNPRUG/C01RMPVUUK1/thread/C01RMPVUUK1-1625672268.339500
```

Comment:
I like the footnote idea for when it's printed out, but there is a more compatible approach to take.

When you are reading a document online, it's better to have links directly go to the new location. The exception is a research site, such as Wikipedia.

When we get to the point of setting this handbook up for printing, that process should include tooling that turns all inline links into footnotes with the full URL.

That means we can write with links inline, for an online reader, and let the rendering take care of how those links appear on the screen or on paper.

Reply:
@quaid - I'm not sure I fully understand this comment. I want to clarify that these are inline links, just a different way of formatting them. But they should render as any normal inline link!

More specifically, when you link using the method i used here (with the number in sq brackets, and then the footnote at the bottom), it appears on the rendered page the same as if you used `[this method](https://somewebsite...)`. At least, that's how it works on the website repo for many of those documents. I actually hadn't known about this format before making some edits to the website, but I found that it made it so much easier to edit the content. While reading and making changes, you aren't distracted by whole link breaking up the sentence. It's especially useful for a page with a lot of links on it.

Reply:
I actually need to change this to [2] instead of (2) or else I'm not sure it will render correctly. But the other link, the one for the github common repo, should be good.

Reply:
Oh that is new to me and pretty cool, [here's the link in the Markdown Guide](1) and that is indeed the legit use, to make the markup easier to read. Sounds like that is already the standard in the project?

[1]: https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
I'm totally on board with this method, I see how it can make the editing much more clear.

One thing is, if we can just keep adding numbers as links regardless of the order in the paragraphs, that would be great (e.g. two paras with links in them using this style are swapped in position, breaking the numerical order.) I would consider making it a specific practice to skip renumbering when numbering goes out of order, if it works otherwise.